graphql-java-support/src/main/proto/reports.proto

```diff
@@ -0,0 +1,488 @@
+syntax = "proto3";
```

Comment:
I think it would be useful to include a comment along the lines of 
```md
This file is copied over from the `apollo-engine-reporting` package, which, in turn, derives its information from Apollo's cloud infrastructure. It is expected that as the protobuf definition changes within the underlying system we are reporting to, these libraries will need to be updated in sync.
```